pkg/kv/bulk/sst_batcher.go

```diff
@@ -251,6 +251,21 @@ func (b *SSTBatcher) updateMVCCStats(key storage.MVCCKey, value []byte) {
 	b.ms.ValCount++
 }
 
+func (b *SSTBatcher) AddMVCCKeyWithImportEpoch(
+	ctx context.Context, key storage.MVCCKey, value []byte, importEpoch uint32,
+) error {
+	mvccVal, err := storage.DecodeMVCCValue(value)
+	if err != nil {
+		return err
+	}
+	mvccVal.MVCCValueHeader.ImportEpoch = importEpoch
+	encVal, err := storage.EncodeMVCCValue(mvccVal)
+	if err != nil {
+		return err
+	}
+	return b.AddMVCCKey(ctx, key, encVal)
```

Comment:
We may want to consider adding a function to decode-modify-encode an MVCCValue (taking a callback), so that we can make use of the faster inlined functions. But we can do that after this PR too.